### DISCUSSION TITLE
Can U Plz Add Suppert Ely.by Accounts  In All Versions

### DISCUSSION THREAD

**Discussion by Dream01018015** [meteor/meteor#13517] [Feature Requests]
Can U Plz Add Suppert Ely.by Accounts  In All Versions ? plz i need it add

**Comment by StorytellerCZ** [comment]
Hello @Dream01018015,

If you are requesting someone to build something it is common courtesy to use proper grammar and describe a bit more of the issue and in posting here reason why it should be included in Meteor core. For example pointing to the OAuth docs: https://docs.ely.by/en/oauth.html

I'm closing this discussion as there is no reason why this should be included in Meteor core. You can easily build this by copying what other OAuth packages have done. Alternatively you can commission package developers like me to build.
Here is how I build some in the past: https://www.youtube.com/watch?v=gRPPF0cw17A
